### ISSUE TITLE
Impossible to change opacity of 'uom' through animations.

### ISSUE THREAD

**Issue by stinobook** [AmoebeLabs/swiss-army-knife-card#217]
## Swiss Army Knife version
2.4.4

## Bug description
Impossible to change opacity of 'uom' through animations. maybe other styles too? 

### To Reproduce

tested on 2 different cards/entities/... 
state works but uom doesn't
```

                        - type: state
                          position:
                            cx: 10
                            cy: 92
                          styles:
                            state:
                              text-anchor: start
                              font-size: 9em
                            uom:
                              opacity: 1 # it is possible to style it here, but not in the animations
                          entity_indexes:
                            - entity_index: 1
                            - entity_index: 0
                          user_actions:
                            tap_action:
                              haptic: success
                              actions:
                                - action: more-info
                          animations:
                            - state: 'on'
                              entity_index: 0
                              styles:
                                state:
                                  opacity: 0.8
                                  transition: opacity 0.5s linear
                                uom:
                                  opacity: 0.8
                                  transition: opacity 0.5s linear
                            - state: 'off'
                              entity_index: 0
                              styles:
                                state:
                                  opacity: 0
                                  transition: opacity 0.5s linear
                                uom:
                                  opacity: 0
                                  transition: opacity 0.5s linear
```
### Expected behavior
for the uom to follow the state animation

### Screenshots
![image](https://github.com/AmoebeLabs/swiss-army-knife-card/assets/51697064/3ebc0da0-86ca-491c-8d89-da3921dc3fd1)
![image](https://github.com/AmoebeLabs/swiss-army-knife-card/assets/51697064/5053bea1-fef7-42c6-bdc1-2a337ff81709)

**Comment by AmoebeLabs** [comment]
I’ll check uom, but in the mean time, try the following:
```yaml
                          animations:
                            - state: 'on'
                              entity_index: 0
                              styles:
                                tool:
                                  opacity: 0.8
                                  transition: opacity 0.5s linear
                            - state: 'off'
                              entity_index: 0
                              styles:
                                tool:
                                  opacity: 0
                                  transition: opacity 0.5s linear
```

**Comment by stinobook** [comment]
I also tried that but it didn't do anything at all. I saw 'tools' once as a possibility in one of your examples and figured maybe it was only 'tools' that worked and not a single 'tool'.


![image](https://github.com/AmoebeLabs/swiss-army-knife-card/assets/51697064/61079785-5666-4445-b212-5f4b81255c80)
![image](https://github.com/AmoebeLabs/swiss-army-knife-card/assets/51697064/74ab3a00-4601-4793-aa82-031974023e8d)
![image](https://github.com/AmoebeLabs/swiss-army-knife-card/assets/51697064/2e1a2e9f-541d-4c7e-9a85-fbe4ad56233a)

**Comment by AmoebeLabs** [comment]
Tool styling is indeed broken for at least the enity state tool. I will check what’s changed. 

Some of the eslint recommendations broke most of my cards while I was changing the development environment. This is either one of those results, or something completely different. It will be solved though. 

Uom styling still unclear. The styleMap for uom is processed by the renderer, so probably the style builder has some error that is causing this bug.

**Comment by AmoebeLabs** [comment]
Closed with #222

**Comment by stinobook** [comment]
This is working great now :D Tool still didnt work but uom and state seperately did the trick.

![image](https://github.com/AmoebeLabs/swiss-army-knife-card/assets/51697064/3d5624d8-d33e-4dc3-a11d-3462afcea35d)
